requirements/torch_cu118_windows.txt

```diff
@@ -1,8 +1,5 @@
 #pip install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu117
 --extra-index-url https://download.pytorch.org/whl/cu118
-cython
-wheel
-pillow==9.3.0
```

Comment:
These 3 should be in common.txt. right? please double check .

Reply:
Yes, confirmed